Add containerized env post

amirfefer opened the pull request.

amirfefer committed:
add containerized app example and fix repo structure

avitova commented:
I went through the doc, and it was all nice and clear! Thanks!

ezr-ondrej reviewed (commented):
This is great, we could maybe bublish this one as the main developer blog post.

Few comments inline :)

amirfefer commented:
Thanks @ezr-ondrej , @avitova !
I unify these two post into one :)
I fixed your comments @ezr-ondrej , and also changed a bit the structure of this repository.

I changed the title of this post, but I'm not sure, I'd like to have some suggestion if you have any
